DESCRIPTION

```diff
@@ -10,6 +10,8 @@ Authors@R: c(
     person("Evgeni", "Chasnovski", email = "evgeni.chasnovski@gmail.com", role = "ctb", comment = c(ORCID = "0000-0002-1617-4019")),
     person("Will", "Hopper", email = "wjhopper510@gmail.com", role = "ctb", comment = c(ORCID = "0000-0002-7848-1946")),
     person("Marium", "Tapal", email = "mariumtapal@gmail.com", role = "ctb", comment = c(ORCID = "0000-0001-5093-6462"))
+    person("Wayne", "Ndlovu", email = "waynedndlovu5@gmail.com", role = "ctb"),
```

Comment:
Dataset is not processed nor the documentation is added

Reply:
@amutaya @agoswa to be added as contributors